Sources/SwiftCrossUI/Values/PresentationDragIndicatorVisibility.swift

```diff
@@ -0,0 +1,3 @@
+public enum PresentationDragIndicatorVisibility: Sendable {
```

Comment:
Apple seems to use [a generic `Visibility` enum](https://developer.apple.com/documentation/swiftui/visibility) for this situation. Probably makes sense to copy them there, cause I can see such an enum being useful in other contexts as well. We should have an `automatic` case as well (which would be whatever the backend's default is).